CMakeLists.txt

```diff
@@ -83,9 +83,9 @@ set(AMICI_SRC_LIST
 add_library(${PROJECT_NAME} ${AMICI_SRC_LIST})
 
 target_link_libraries(${PROJECT_NAME}
-    ${SUNDIALS_LIB_DIR}/libsundials_nvecserial${CMAKE_SHARED_LIBRARY_SUFFIX}
-    ${SUNDIALS_LIB_DIR}/libsundials_cvodes${CMAKE_SHARED_LIBRARY_SUFFIX}
-    ${SUNDIALS_LIB_DIR}/libsundials_idas${CMAKE_SHARED_LIBRARY_SUFFIX}
+    ${SUNDIALS_LIB_DIR}/libsundials_nvecserial${CMAKE_STATIC_LIBRARY_SUFFIX}
+    ${SUNDIALS_LIB_DIR}/libsundials_cvodes${CMAKE_STATIC_LIBRARY_SUFFIX}
+    ${SUNDIALS_LIB_DIR}/libsundials_idas${CMAKE_STATIC_LIBRARY_SUFFIX}
```

Comment:
Why only static linking of sundials and not of suitesparse?

Reply:
the build scripts don't generate static versions of those libraries yet ;)